### DISCUSSION TITLE
Is PrimeReact useable in Ignite

### DISCUSSION THREAD

**Discussion by bodeg** [infinitered/ignite#1793] [Q&A] [answered]
This might be a very stupid question...please, allow me...

Can I use PrimeReact components with React Native, especially with Ignite 

https://www.primefaces.org/primereact/

**Comment by jamonholmgren** [comment, marked as answer]
It doesn't appear so! (Not a stupid question, though!)

https://github.com/primefaces/primereact/issues/811#issuecomment-477544892
